README.md

```diff
@@ -2,15 +2,31 @@
 
 C++ Idiomatic Clients for [Google Cloud Platform][cloud-platform] services.
 
-[![Travis CI status][travis-shield]][travis-link]
+<!-- Core Results -->
```

Comment:
Would these be nice as subheadings instead of comments?

Reply:
Using actual headers (the markdown equivalent of h2, h3, etc) looked bad (to me), I used bolded words:

https://github.com/coryan/google-cloud-cpp/blob/update-README-shields/README.md

Reply:
Optional: Perhaps stick them in a little table like on the internal team page?

Reply:
I did try tables:

https://github.com/coryan/google-cloud-cpp/blob/5257e43b1e9afc18305350552b5e18b09ffee183/README.md

Did not like how much horizontal space they took.  But if we can figure out a way to make them more compact I would be all for them.  Keep in mind that the README file is the first file folks go to for information about "what is this?", "how do I compile it?", etc.

Reply:
Ack.  Let's leave any possibility of compact tables for another day.  Thanks.